packages/common/src/core/schemas/internal/follow/FollowCardSchema.ts

```diff
@@ -0,0 +1,44 @@
+import { IConfigurationSchema } from "../../types";
```

Comment:
I might suggest naming the following schema something more generic, like `FollowSchema` or `FollowActionSchema` - as we discussed earlier this week, there may be other contexts (that aren't a card) in which we allow you to configure a follow action, and we should reuse this schema. The `uiSchemas` are where we can get more context-specific

Reply:
Good point, I'll change this to a more generic name